src/conduit/article/views.py

```diff
@@ -21,6 +21,22 @@
 SingleArticleResponse = TypedDict("SingleArticleResponse", {"article": Article})
 
 
+@view_config(route_name="feed", renderer="json", request_method="GET", openapi=True)
+def feed(request: Request) -> MultipleArticlesResponse:
+    """Get your article feed."""
+    q = request.db.query(Article)
+    q = q.order_by(desc("created"))
```

Comment:
Usually, order comes as last in SQL

Reply:
Nah, won't work:

```
sqlalchemy.exc.InvalidRequestError: Query.order_by() being called on a Query which already has LIMIT or OFFSET applied. To modify the row-limited results of a  Query, call from_self() first.  Otherwise, call order_by() before limit() or offset() are applied.
```